YetAnotherWeather/UserStories/Presentation/WeatherDetails/Views/CurrentWeatherView.swift

```diff
@@ -117,19 +101,22 @@ final class CurrentWeatherView: UIView {
 
 extension CurrentWeatherView: ConfigurableView {
     
-//    struct Model {
-//        let location: String
-//        let currentTemp: String
-//        let conditions: String
-//        let minTemp: String
-//        let maxTemp: String
-//    }
+    struct Model {
+        let location: String
+        let currentTemp: String?
+        let conditions: String
+        let minTemp: String?
+        let maxTemp: String?
+    }
     
-    func configure(with model: WeatherDetailsViewModel.CurrentWeatherViewModel) {
-        self.backgroundColor = UIColor.systemGray3
+    func configure(with model: Model) {
         locationLabel.text = model.location
         currentTempLabel.text = model.currentTemp
         conditionsLabel.text = model.conditions
-        highAndLowLabel.text = "H:\(model.maxTemp)  L:\(model.minTemp)"
+        guard let maxTemp = model.maxTemp, let minTemp = model.minTemp else {
+            highAndLowLabel.isHidden = true
+            return
+        }
+        highAndLowLabel.text = "H:\(maxTemp)  L:\(minTemp)"
```

Comment:
guard в середине метода старайся не писать
```suggestion
        if let maxTemp = model.maxTemp, let minTemp = model.minTemp {
            highAndLowLabel.text = "H:\(maxTemp)  L:\(minTemp)"
            highAndLowLabel.isHidden = false
        } else {
            highAndLowLabel.isHidden = true
        }
```